src/getHumanAge.test.js

```diff
@@ -8,6 +8,74 @@ describe(`Function 'getHumanAge':`, () => {
   });
 
   it(`should return an array`, () => {
+    const result = getHumanAge(15, 16);
 
+    expect(result).toBeInstanceOf(Array);
+  });
+
+  it(`shoud return an array [0, 0] when 'catAge' and 'dogAge' < 15 `, () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  it(`shoud return an array [1, 0] 
+  when 'catAge' = 15 and 'dogAge' < 15 `, () => {
+    expect(getHumanAge(15, 14)).toEqual([1, 0]);
+  });
+
+  it(`shoud return an array [0, 1] 
+  when 'catAge' < 15 and 'dogAge' = 15 `, () => {
+    expect(getHumanAge(14, 15)).toEqual([0, 1]);
+  });
+
+  it(`shoud return an array [1, 1] 
+  when 'catAge' = 15 and 'dogAge' = 15 `, () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
+  it(`shoud return an array [1, 1] 
+  when 'catAge' and ('dogAge' > 15 && 'dogAge' < 24 )`, () => {
```

Comment:
Try to describe your tests without extra parentheses and chars like '&&', it should not look like a part of JS code :)

Reply:
fixed